recipes/libyaml/all/conanfile.py

```diff
@@ -0,0 +1,68 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class LibYAMLConan(ConanFile):
+    name = "libyaml"
+    description = "LibYAML is a YAML parser and emitter library."
+    topics = ("conan", "libyaml", "yaml", "parser", "emitter")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/yaml/libyaml"
+    license = "MIT"
+    exports_sources = ["CMakeLists.txt"]
```

Comment:
Same, not sure if needed.

Reply:
Yes, without it, that CMakefile won't be copied to source folder.